osc/core.py

```diff
@@ -5603,11 +5603,13 @@ def get_repositories(apiurl):
     return r
 
 
-def get_distibutions(apiurl, discon=False):
-    r = []
+def get_distributions(apiurl, discon=False):
+    """Returns list of dicts with headers
+      'distribution', 'project', 'repository', 'reponame'"""
```

Comment:
This doesn't apply to the `DISCONTINUED` distributions below.
Could you rewrite the code to return list of dicts too?

Reply:
The option to list discontinued distributions was commented out about 9 years ago.

https://github.com/openSUSE/osc/blame/5d7bd223400b181b061935a871faf6db9355d1d4/osc/commandline.py#L5329

So I propose to just remove this block.

Reply:
Agreed.
Could you remove the `discon=False` argument and the whole related block of code then?

Reply:
Removed and rebased.